hackle-sdk-core/src/main/kotlin/io/hackle/sdk/core/evaluation/evaluator/inappmessage/InAppMessageFlowEvaluator.kt

```diff
@@ -154,17 +162,74 @@ internal class HiddenInAppMessageFlowEvaluator(
  * IAM 타겟팅이 된 경우
  */
 internal class TargetInAppMessageFlowEvaluator(
-    private val targetMatcher: InAppMessageTargetMatcher,
-    private val inAppMessageResolver: InAppMessageResolver
+    private val targetMatcher: InAppMessageTargetMatcher
 ) : InAppMessageFlowEvaluator {
     override fun evaluate(
         request: InAppMessageRequest,
         context: Evaluator.Context,
         nextFlow: InAppMessageFlow
-    ): InAppMessageEvaluation {
-        if (targetMatcher.matches(request, context)) {
-            return inAppMessageResolver.resolve(request, context, IN_APP_MESSAGE_TARGET)
+    ): InAppMessageEvaluation? {
+        val isTargetMatched = targetMatcher.matches(request, context)
+        if (isTargetMatched) {
+            return nextFlow.evaluate(request, context)
         }
         return InAppMessageEvaluation.of(request, context, NOT_IN_IN_APP_MESSAGE_TARGET)
```

Comment:
```suggestion
        val isTargetMatched = targetMatcher.matches(request, context)
        if (!isTargetMatched) {
            return InAppMessageEvaluation.of(request, context, NOT_IN_IN_APP_MESSAGE_TARGET)
        }
        return nextFlow.evaluate(request, context)
```

이렇게 바꾸는게 통일성 있을 것 같습니다